src/goInstallTools.ts

```diff
@@ -115,9 +115,20 @@ function installTools(goVersion: SemVersion, missing?: string[]) {
 
 	outputChannel.appendLine(''); // Blank line for spacing.
 
+	// http.proxy setting takes precedence over environment variables
+	let httpProxy = vscode.workspace.getConfiguration('http').get('proxy');
+	let env = process.env;
+	if (httpProxy) {
+		env = Object.assign({}, process.env, {
+			http_proxy: httpProxy,
```

Comment:
I found the failed test case is `Test diffUtils.getEdits` however I have no idea how does this commit make a difference on that. I'll have a local debug to see what happened.